highfidelity/hifi
# Case 21564: Warn When Muted

# TEST PLAN
- Launch Interface in HMD mode
- mute yourself
- start talking
- there should be text in the center of your screen at the bottom indicating you are muted
- switch to desktop mode
- mute yourself
- start talking
- there should be text in the center of your screen at the bottom indicating you are muted
- going into the audio app in the tablet and deselecting 'warn when muted' will cause the text indicator not to show up in desktop nor in HMD mode

https://highfidelity.manuscript.com/f/cases/21564/Warn-When-Muted
